modules/gapi/misc/python/samples/gaze_estimation.py

```diff
@@ -0,0 +1,403 @@
+                                             cv.GArray.GMat],
+            out_types=[cv.GArray.GMat])
+class ProcessPoses:
+    @staticmethod
+    def outMeta(arr_desc0, arr_desc1, arr_desc2):
+        return cv.empty_array_desc()
+
+
+@cv.gapi.kernel(ProcessPoses)
+class GProcessPosesImpl:
+    @staticmethod
+    def run(in_ys, in_ps, in_rs):
+        out_poses = []
+        size = len(in_ys)
+        for i in range(size):
+            out_poses.append(np.array([in_ys[i][0], in_ps[i][0], in_rs[i][0]]).T)
+        return out_poses
+
+
+@cv.gapi.op('custom.ParseEyes', in_types=[cv.GArray.GMat,
+                                          cv.GArray.Rect,
+                                          cv.GOpaque.Size],
+            out_types=[cv.GArray.Rect,
+                       cv.GArray.Rect,
+                       cv.GArray.Point,
+                       cv.GArray.Point])
+class ParseEyes:
+    @staticmethod
+    def outMeta(arr_desc0, arr_desc1, arr_desc2):
+        return cv.empty_array_desc(), cv.empty_array_desc(), \
+               cv.empty_array_desc(), cv.empty_array_desc()
+
+
+@cv.gapi.kernel(ParseEyes)
+class GParseEyesImpl:
+    @staticmethod
+    def run(in_landm_per_face, in_face_rcs, frame_size):
+        left_eyes = []
+        right_eyes = []
+        midpoints = []
+        lmarks = []
+        num_faces = len(in_landm_per_face)
+        surface = (0, 0, *frame_size)
+        for i in range(num_faces):
+            rect = in_face_rcs[i]
+            points = process_landmarks(*rect, in_landm_per_face[i])
+            for p in points:
+                lmarks.append(p)
+            size = int(len(in_landm_per_face[i][0]) / 2)
+
+            rect, midpoint_l = eye_box(lmarks[0 + i * size], lmarks[1 + i * size])
+            left_eyes.append(intersection(surface, rect))
+            rect, midpoint_r = eye_box(lmarks[2 + i * size], lmarks[3 + i * size])
+            right_eyes.append(intersection(surface, rect))
+            midpoints += [midpoint_l, midpoint_r]
+        return left_eyes, right_eyes, midpoints, lmarks
+
+
+@cv.gapi.op('custom.GetStates', in_types=[cv.GArray.GMat, cv.GArray.GMat],
+            out_types=[cv.GArray.Int, cv.GArray.Int])
+class GetStates:
+    @staticmethod
+    def outMeta(arr_desc0, arr_desc1):
+        return cv.empty_array_desc(), cv.empty_array_desc()
+
+
+@cv.gapi.kernel(GetStates)
+class GGetStatesImpl:
+    @staticmethod
+    def run(eyesl, eyesr):
+        size = len(eyesl)
+        out_l_st = []
+        out_r_st = []
+        for i in range(size):
+            for st in eyesl[i]:
+                out_l_st += [1 if st[0] < st[1] else 0]
+            for st in eyesr[i]:
+                out_r_st += [1 if st[0] < st[1] else 0]
+        return out_l_st, out_r_st
+
+# FIXME: the operation should be wrapped soon
+@cv.gapi.op('custom.Copy', in_types=[cv.GMat], out_types=[cv.GMat])
+class Copy:
+    @staticmethod
+    def outMeta(desc):
+        return desc
+
+
+@cv.gapi.kernel(Copy)
+class GCopyImpl:
```

Comment:
Why don't you just wrap this one on your own ? 
It should be pretty straightforward

Reply:
Copy is wrapped